### DISCUSSION TITLE
[SOLVED] I don’t received benefits.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63650] [GitHub Education]
My Student Developer Pack is approved but i don't received any benefit. How can i receive?

EDIT: I received benefits after 3-4 days, now there is no problem. Thanks to readers.

**Comment by naveedalirehmani** [comment]
1. Check your email, including the spam folder.
2. Visit the Student Developer Pack page to see the list of benefits and redeem them.

**Reply by ghost** [reply]
> 1. Check your email, including the spam folder.
> 
> 2. Visit the Student Developer Pack page to see the list of benefits and redeem them.

No email arrived, but i can see in site my request is approved.
